web_deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+if [ "$TRAVIS_BRANCH" == "master" ]; then # work just for master
```

Comment:
Notice that this also triggers on pull requests targeting that branch. You may also want to use TRAVIS_PULL_REQUEST. See the comment of the first answer at [1]

[1] https://stackoverflow.com/questions/31338562/travisci-run-after-success-on-a-specific-branch

Reply:
f490699